common-items.lic

```diff
@@ -1002,12 +1002,33 @@ module DRCI
   # GIVE/ACCEPT ITEM
   #########################################
 
-  def give_item?(target, item)
+  def give_item?(target, item = nil)
+    command = item ? "give my #{item} to #{target}" : "give #{target}"
     # An offer expires after 30 seconds.
     # The timeout is set to 35 to give some buffer for clock skew.
-    case DRC.bput("give my #{item} to #{target}", { 'timeout' => 35 }, "has accepted your offer", "has declined the offer", "Your offer to .* has expired", "You may only have one outstanding offer at a time", "What is it you're trying to give", "There's nothing in your right hand to give away")
-    when "has accepted your offer"
+    # Expanded to include repair tickets
+    case DRC.bput(command, { 'timeout' => 35 }, 
+            "has accepted your offer", "has declined the offer", "You don't need to specify the object",
+            "Your offer to .* has expired", "You may only have one outstanding offer at a time", "What is it you're trying to give",
+            "There's nothing in your right hand to give away", 'Just give it to me again', "If you agree, give it to me again",
+            "You don't need to specify the object", "^You hand.*gives you back a repair ticket", "Please don't lose this ticket!", 
+            "I don't repair those here", "What is it", "There isn't a scratch on that", "I will not", "I can't fix those",
+            "^You hand.*your ticket and are handed back")
```

Comment:
```suggestion
            /^You hand .* your ticket and are handed back/
```
I won't do them all, but for anchors to work you need regex syntax.

Reply:
Sorry, I made this comment last week, but clicked on "start a review" instead of single comment, so it didn't get published...

Reply:
Gotcha, ty.